### ISSUE TITLE
Allow mutliple triplets for install command

### ISSUE THREAD

**Issue by smkanadl** [lukka/CppBuildTasks#12]
I use vcpkg to create native nuget packages. For some of them I package both (x86 and x64) flavours. From looking at the source, I figured out that the triplet parser only reports the first occurance of --triplet. But I would like to use run-vcpkg in a way like this

```
libmodbus
--triplets
x86-windows
x64-windows
```
or like this
```
libmodbus
--triplet
x86-windows
libmodbus
--triplet
x64-windows
```
vcpkg command line already supports this:
```
libmodbus:x86-windows
libmodbus:x64-windows
```

Currently I tried to use two run-vcpkg steps, but this seems not to be optimal. It also seems to have issues when being combined with the Cache functionality.

Or am I getting something wrong?

**Comment by lukka** [comment]
@smkanadl 
have not tried the scenario with multiple triplets yet, but it could just be working already.

Could you verify whether a response like this would work:

```
sqlite3:x64-osx
sqlite3:x86-osx
libmodbus:x86-osx
libmodbus
zlib
--triplet
x64-osx
```

For all the ports that have no specified triplet, the one specified by `--triplet` will be used, that is you will get installed the following ports:

```
libmodbus:x64-osx
libmodbus:x86-osx
sqlite3:x64-osx
sqlite3:x86-osx
zlib:x64-osx   
```

If this is not working, let's try to propose a possible solution. All I have in mind right now is that the `run-vcpkg` task should not enforce any '--triplet' argument unless it is provided in the `vcpkgTriplet` task parameter (so it is up to the user to pass it or not).

**Comment by smkanadl** [comment]
@lukka Thanks for your support.

I tried 
```
libmodbus:x86-windows
libmodbus:x64-windows
```
and it works just as I expected. It picked up `x86-windows` as default triplet (because I provided non).

I am not sure why I did not tried it out directly. Probably I was a bit confused by you examples as they all used the syntax with one triplet. Enhancing the documentation by some response files would be good.
